packages/kmod-6.1-nvidia-r570/kmod-6.1-nvidia-r570.spec

```diff
@@ -47,13 +47,17 @@ Source206: nvidia-persistenced.service
 # NVIDIA tesla conf files from 300 to 399
 Source300: nvidia-tesla-tmpfiles.conf
 Source301: nvidia-tesla-build-config.toml.in
-Source302: nvidia-open-gpu-config.toml.in
-Source303: nvidia-open-gpu-copy-only-config.toml.in
-Source304: nvidia-ld.so.conf.in
-Source305: link-tesla-kernel-modules.service.in
-Source306: load-tesla-kernel-modules.service.in
-Source307: copy-open-gpu-kernel-modules.service.in
-Source308: load-open-gpu-kernel-modules.service.in
+Source302: nvidia-ld.so.conf.in
+Source303: nvidia-open-gpu-config.toml.in
+Source304: nvidia-open-gpu-copy-only-config.toml.in
```

Comment:
Why do these end up in a different order?

Reply:
My last rebase unwound this, I'll correct it.

Reply:
Moved these around to avoid future changes, they have new starting numbers to avoid more refactors in the future.